src/com/mendix/widget/carousel/components/Carousel.ts

```diff
@@ -0,0 +1,27 @@
+import { DOM, createElement } from "react";
+
+import { CarouselItem } from "./CarouselItem";
+
+interface Image {
+    imageUrl?: string;
+}
+
+export interface CarouselProps {
+    images?: Image[];
+}
+
+const carouselItems = (images: Image[]) => {
+    return images.map((image, index) => createElement(CarouselItem, {
+        active: index === 0 ? true : false,
+        imageUrl: image.imageUrl,
+        key: index
+    }));
+};
+
+export const Carousel = (props: CarouselProps) => {
+    return (DOM.div({ className: "carousel" },
+        DOM.div({  className: "carousel-inner" },
```

Comment:
Removed.
For some frustrating reason, no IDE or linter is catching this, yet they catch any extra spaces before the closing brace.